e2e/playwright/create-actions/src/tests/create-library.e2e.ts

```diff
@@ -49,8 +49,10 @@ test.describe('Create Libraries ', () => {
   let randomLibraryId: string;
   let randomLibraryDescription: string;
   const libraryDialogTitle = 'Create Library';
-  const libraryNameLabel = 'Name *';
-  const libraryIdLabel = 'Library ID *';
+  // toDo add in adf * to the placeholder
```

Comment:
Do we have a ticket for that? Is someone working on it?

Reply:
added related tickets, will also update description for them